src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -37,6 +38,13 @@ def load_arguments(self, _):
         c.argument('no_proxy', options_list=['--proxy-skip-range'], help='List of URLs/CIDRs for which proxy should not to be used.')
         c.argument('proxy_cert', options_list=['--proxy-cert'], type=file_type, completer=FilesCompleter(), help='Path to the certificate file for proxy')
         c.argument('disable_proxy', options_list=['--disable-proxy'], action='store_true', help='Disables proxy settings for agents')
+        c.argument('auto_upgrade', options_list=['--auto-upgrade'], help='Flag to enable/disable auto upgrade of arc agents. By default, auto upgrade of agents is enabled.', arg_type=get_enum_type(["true", "false"]))
```

Comment:
Here in update, we want toggling experience for user, so I am thinking to keep it here as a flag to enable/disable. Let me know, what you think.